src/API.CRUD/Extensions/JsonStatusCodePagesExtensions.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Diagnostics;
+using Microsoft.AspNetCore.Http;
+
+namespace API.CRUD.Extensions
+{
+    public static class JsonStatusCodePagesExtensions
+    {
+        private const string JsonContentType = "application/json";
+
+        /// <summary>
+        /// Status code pages are rendered as JSON compatible responses.
+        /// </summary>
+        /// <param name="app">A class that provides the mechanisms to configure an application's request.</param>
+        /// <returns>This class that provides the mechanisms to configure an application's request</returns>
+        public static IApplicationBuilder JsonStatusCodePages(this IApplicationBuilder app)
+        {
+            if (app == null)
+            {
+                throw new ArgumentNullException(nameof(app));
+            }
+
+            return app.UseStatusCodePages(OnStatusCodePagesAsync);
+        }
+
+        private static Task OnStatusCodePagesAsync(StatusCodeContext context)

```

Comment:
Does this need to be a seperate method? Not used by anything other than above method, and seems pretty trivial..maybe just put the code in the method above?

Reply:
Change to inline.

Reply:
Leaving as a method instead of line. Looks cleaner/easier to grok.